### DISCUSSION TITLE
26F9FF8A-21F5-401A-9DCC-C25334F294A1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#112] [General]
# 26F9FF8A-21F5-401A-9DCC-C25334F294A1

A Rotovator is is a spinning tether designed to take craft from a planetary surface to escape velocity in one easy step. This one is around Mars. There are plenty of arranged shuttle encounters and a few hair raising near-collisions. Three orbiting hanging tethers are included as well.
V2.0 (February, 2011): contains additional features.

https://celestia.mobi/addon?item=26F9FF8A-21F5-401A-9DCC-C25334F294A1

**Comment by EricNelson-sketch** [comment]
It's too bad there's no date to when this is invented. We know for sure it doesn't exist yet, and it's very unrealistic to have it set to the present.
But great addon nonetheless.
